### PR TITLE
Jdk252

### PR DESCRIPTION
**Pull request by asheppard-work**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ pipeline {
 		timestamps()
 
 		// Prevent concurrent builds
-		disableConcurrentBuilds();
+		disableConcurrentBuilds()
 	}
 	stages {
 		stage ('1.3.3 Build') {
```
Comment (hsmith-gresham) [inline Jenkinsfile]: The version slightly further down here will need updating to the next version before this gets merged as per the readme for releasing

**Review by hsmith-gresham — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Unreleased:
 - Added a password encoder to the client entity service
 - Fixes a bug by specifying the name of the scope columnn
 - Removed functionality that passed the client secret down to the UI
+- Updated JDK to Corretto 1.8.252
```
Comment (gresham-Carling) [inline CHANGELOG.md:9]: I'm not sure there's much worth maintaining the change log here given we don't have anything in other products.

**Review by gresham-Carling — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (asheppard-work) [reply CHANGELOG.md:9]: We've been maintaining a changelog in some of our newer and smaller repos and it's been quite useful.
I don't plan to make any further changes in this repo!

File: Jenkinsfile (hunk 1 above)
Comment (hsmith-gresham) [reply Jenkinsfile]: Scrap this - the releaseImprovements branch has been merged so this branch can just be updated and it should be good to just merge in

**Review by hsmith-gresham — APPROVED** [review]

**Pull request merged**